src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +21,15 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+
+        return numbers.stream()
+                .flatMap(str -> Arrays.stream(str.split(COMMA)))
+                .map(Integer::valueOf)
+                .filter(num -> num % 2 == 0)
+                .sorted()
+                .findFirst()
```

Comment:
You can minimize stream operations, but this is just a recommendation
```suggestion
                .mapToInt(Integer::parseInt)
                .filter(num -> num % 2 == 0)
                .min()
```